xml/System.Text/StringBuilder.xml

```diff
@@ -3925,7 +3892,31 @@ The `Equals` method performs an ordinal comparison to determine whether the char
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>GetChunks returns ChunkEnumerator that follows the IEnumerable pattern and
+             thus can be used in a C# 'foreach' statements to retreive the data in the StringBuilder
+             as chunks (ReadOnlyMemory) of characters.  An example use is:
+             
+                  foreach (ReadOnlyMemory&lt;char&gt; chunk in sb.GetChunks())
+                     foreach(char c in chunk.Span)
+                         { /* operation on c }
+            
+             It is undefined what happens if the StringBuilder is modified while the chunk
+             enumeration is incomplete.  StringBuilder is also not thread-safe, so operating
+             on it with concurrent threads is illegal.  Finally the ReadOnlyMemory chunks returned 
+             are NOT guarenteed to remain unchanged if the StringBuilder is modified, so do 
+             not cache them for later use either.  This API's purpose is efficiently extracting
+             the data of a CONSTANT StringBuilder.  
+             
+             Creating a ReadOnlySpan from a ReadOnlyMemory  (the .Span property) is expensive 
+             compared to the fetching of the character, so create a local variable for the SPAN 
+             if you need to use it in a nested for statement.  For example 
+             
+                foreach (ReadOnlyMemory&lt;char&gt; chunk in sb.GetChunks())
+                {
+                     var span = chunk.Span;
+                     for(int i = 0; i &lt; span.Length; i++)
+                         { /* operation on span[i] */ }
+                }</summary>
```

Comment:
Most of this should not be in summary.

Reply:
Oof, good catch. I'll leave the essential description in the summary, and move the rest to the Remarks.

Reply:
Fixed. I'll wait for @mairaw / @rpetrusha (and a successful build) to ensure this change was done correctly.